### DISCUSSION TITLE
part 2, double sort

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#116] [Tarea 1] [answered]
Hola en el evento DOUBLE-SORT-BY, podemos asumir que type1 y type2 serán distintos?? es decir, no existirán casos como:
DOUBLE-SORT-BY age asc age desc?

**Comment by Segismundios** [comment, marked as answer]
Holaa @GuilleStanke, puedes asumir que serán distintos.
